consume-more-stuff/server/routes/items.js

```diff
@@ -30,15 +28,14 @@ router.post("/", (req, res) => {
     title
   } = req.body;
 
-  console.log('items req body', req.body);
-
-  //const parsedId = parseInt(id);
   const parsedCat = parseInt(category_id);
   const parsedCond = parseInt(condition_id);
   const parsedStat = parseInt(status_id);
 
+  if (title.length <= 3 || description.length <= 10) {
```

Comment:
I'm ok with this but you could break these out as individual checks and return more detailed information on which fields are wrong. More UX than right vs. wrong.